src/libs/routes/errorHandler.ts

```diff
@@ -1,4 +1,4 @@
-export const notFoundRouteMiddleware = (err , req , res, next ) => {
+export const errorHandlerMiddleware = (err , req , res, next ) => {
    console.log('INSIDE 2nd MIDDLEWARE', err);
    res.send( {
        error: 'Not Found',
```

Comment:
@rohit-ns This will always throw 'Not Found' as error, please update it accordingly.

Reply:
done please review